python/test/unit/function/test_expression_evaluation.py

```diff
@@ -200,8 +201,7 @@ def test_assembly_into_quadrature_function():
     mesh = dolfinx.UnitSquareMesh(MPI.COMM_WORLD, 3, 6)
 
     quadrature_degree = 2
-    quadrature_points = FIAT.create_quadrature(FIAT.ufc_simplex(
-        mesh.topology.dim), quadrature_degree, scheme="default").get_points()
+    quadrature_points = basix.make_quadrature(basix.CellType.triangle, quadrature_degree)
```

Comment:
Even if this is skipped, this is not the correct way of calling `make_quadrature`, ref: https://github.com/FEniCS/basix/blob/main/cpp/quadrature.h#L66

Reply:
I've created an issue to remind me to fix this test once quadrature elements are fixed: https://github.com/FEniCS/ffcx/issues/298